pylxd/exceptions.py

```diff
@@ -1,2 +1,10 @@
 class ClientConnectionFailed(Exception):
     """An exception raised when the Client connection fails."""
+
+
+class NotFound(Exception):
+    """Generic get failure exception."""
+
+
+class CreateFailed(Exception):
+    """Generic create failure exception."""
```

Comment:
Oh, no parent module-level exception ? I thought it was a best practice and helped in exception handling ...